pkg/prometheus/operator_test.go

```diff
@@ -264,3 +265,49 @@ func TestValidateRemoteWriteConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestRelabelConfig(t *testing.T) {
+	for _, tc := range []struct {
+		relabelConfig monitoringv1.RelabelConfig
+		expectedErr   bool
+	}{
+		{
```

Comment:
could you add a few more test cases and a comment at the top of each case to explain what we're testing?

Reply:
I added some more